modules/ROOT/pages/tutorials/bi.adoc

```diff
@@ -47,19 +47,26 @@ Refer to the link:https://help.tableau.com/current/pro/desktop/en-us/examples_ot
 After downloading the JDBC Neo4j Connector for BI from the https://neo4j.com/download-center/#integrations[Download Center]:
 
 . Close any running instances of Tableau Desktop.
-. Copy the Neo4j driver to the appropriate Tableau drivers folder (for example `C:\Program Files\Tableau\Drivers` on Windows, or `~/Library/Tableau/Drivers` on macOS).
+. Copy the Neo4j driver JAR file into the appropriate Tableau `Drivers` folder.
+* Use `C:\Program Files\Tableau\Drivers` on Windows.
+* Use `~/Library/Tableau/Drivers` on macOS.
+If the folder is not immediately visible, select `Go -> Go to Folder` in the Finder application to open the folder manually.
+
 . Start Tableau and search for the `Other Databases (JDBC)` option.
 . Insert the Aura URL as `jdbc:neo4j://xxxxxxxx.databases.neo4j.io?SSL=true`, leave the SQL dialect as `SQL92`, and complete the relevant credentials.
 
-If the connection fails with a `Generic JDBC connection error`, you can do one of the following:
+After the connection is established, you can select the `neo4j` database and the `Node` schema to find the `Person` table.
```

Comment:
But this is in case the user's database has nodes labeled as Person. If it's a dataset about something else or with different labels, that would be also different, right? Maybe it's also the case to mention that you need a database with data in it in order to be able to see anything, not an empty database.

Reply:
This is already covered in the second paragraph, "Preparing example data". Isn't it clear enough?